WooCommerce/UITestsFoundation/XCTest+Extensions.swift

```diff
@@ -113,4 +113,9 @@ extension XCUIElement {
 
         startCoordinate.press(forDuration: 0.01, thenDragTo: destination)
     }
+
+    public func getTextVisibilityCount(text: String) throws -> Int {
```

Comment:
Moving this here as it's being used multiple times. To be honest, not sure if this is the best place so I'm open to moving this elsewhere.